backend/dashboard/views/components/forms/StyledInput.vue

```diff
@@ -1,20 +1,34 @@
 <template lang="pug">
-label.inputgroup
+label.inputgroup(:class='{ "is-disabled": disabled }')
   span.input-label(v-if='label') {{ label }}
+
+  char-limit-indicator.c-char-len(
```

Comment:
o4-mini-high:

- In **StyledInput.vue**, your scoped `.c-char-len { position:absolute }` won’t actually apply to the inner `<span>` of the child component because of Vue’s scoped CSS markers.  You either need a deep selector (`::v-deep .c-char-len`) or move the positioning into the child component’s own `<style>`.

Reply:
This is incorrect. Just changed the class name to clear the confusion here.